ui/app/ducks/swaps/swaps.js

```diff
@@ -462,13 +462,7 @@ export const signAndSendTransactions = (history, metaMetricsEvent) => {
       conversionRate,
       numberOfDecimals: 6,
     })
-    const averageSavings = usedQuote.isBestQuote
-      ? decEthToConvertedCurrency(
```

Comment:
We are also removing the conversion from ETH to local currency, so that all our savings data can be denominated in the same currency.

Reply:
Also so that we aren't using the wrong conversion rate - this code was incorrect before, as it used `conversionRate` for USD despite `conversionRate` referring to the local currency's conversion rate, not USD.